Fluid.Benchmarks/Fluid.Benchmarks.csproj

```diff
@@ -5,6 +5,12 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Is this needed?!!

Reply:
Unrelated to your PR, but saw this folder shouldn't be included.